basic_autonomy/src/basic_autonomy.cpp

```diff
@@ -460,9 +460,9 @@ namespace basic_autonomy
             output.push_back(all_sampling_points_line1);
             output.push_back(all_sampling_points_line2);
 
-            ros::WallTime end_time = ros::WallTime::now();  // Planning complete
+            ros::Time end_time = ros::Time::now();  // Planning complete
```

Comment:
I think we want to keep these execution timers as Walltime, since that's what we'd actually like to measure (how long it actually takes to compute these values for performance reasons). I don't think the simulation timer will be ticking at a fine enough resolution anyway, we're probably looking at nano-seconds up to a very small handful of milliseconds for this function.